docs/modules/ROOT/pages/utils/_class_hashes.adoc

```diff
@@ -5,6 +5,7 @@
 :Account-class-hash: 0x01148c31dfa5c4708a4e9cf1eb0fd3d4d8ad9ccf09d0232cd6b56bee64a7de9d
 :ERC20-class-hash: 0x07d94f28156c0dc3bfd9a07ca79b15b8da2b5b32093db79000fcd0f6f625d213
 :ERC721-class-hash: 0x06b7c9efc5467c621f58d87995302d940a39b7217b5c5a7a55555c97cabf5cd8
+:ERC1155-class-hash: 0x4bfd228c408fbdd2b1304af525722c8532c47fab33002fa859f2ba24cd9adbd
```

Comment:
```suggestion
:ERC1155-class-hash: 0x04bfd228c408fbdd2b1304af525722c8532c47fab33002fa859f2ba24cd9adbd
```

Reply:
I'm receiving `0x518be7d9fa527c78d6929bf9e638e9c98b6077722e27e9546cc4342e830386e` with the latest changes

Reply:
We still need to pad it for consistency with the other hashes i.e. `0x0518be7...`

Reply:
remember we shouldn't focus too much on this since it should be covered by the CI